### ISSUE TITLE
Keyboard nav focus gets lost in iFrame

### ISSUE THREAD

**Issue by rea-laura** [phetsims/coulombs-law#74]
<b>Test Device</b>

Jordan (8B)

<b>Operating System</b>

Mac Mojave 10.14

<b>Browser</b>

Safari 12.0 (14606.1.36.1.9)

<b>Problem Description</b>
For https://github.com/phetsims/QA/issues/210
Full screen in iFrame allows the keyboard nav focus to be lost off screen and cannot be recovered without using the mouse. 

<b>Steps to Reproduce</b>

1) open sim in iFrame
2) go to full screen mode
3) press tab until keyboard nav focus is off the screen
4) focus cannot be recovered by pressing any keys or by going backwards (shift+tab), but can be recovered by using the mouse to interact with the sim, then pressing tab again.

**Comment by KatieWoe** [comment]
Exploring on a few other platforms.  The keyboard nav loops around once, but upon the second time it disappears on Firefox.  Have not seen issue on chrome. Edge also seems to display the problem.

**Comment by KatieWoe** [comment]
Seen in https://phet-dev.colorado.edu/html/resistance-in-a-wire/1.5.0-rc.9/phet/resistance-in-a-wire_en_iframe_phet.html when testing on firefox, though pressing shift + tab would highlight the phet menu and nothing else.  This seems to mean it is not coulombs-law specific.  @jessegreenberg or @mbarlow12  if you want this issue moved to a new repo.

**Comment by jessegreenberg** [comment]
Hmm, verified in Edge and FF.

**Comment by jessegreenberg** [comment]
In FF, when leaving full screen mode tab navigation is still broken.

EDIT: ...somteimes.

EDIT: In FF, it allows two full loops through keyboard navigation, then it gets stuck.

**Comment by jessegreenberg** [comment]
Problem goes away when the dev tools are open.

**Comment by jessegreenberg** [comment]
Wait, this sounds crazy but in FF, it seems related to the popup that FF shows when entering full screen mode. When you first enter full screen mode, FF pops up a little dialog that says "localhost:8080 is now full screen". As long as that is showing, you can tab navigate all you want. Once it disappears, tab navigation in full screen mode is broken.

**Comment by jessegreenberg** [comment]
In FF I added this: 
```js
window.setInterval( function() {
  console.log( document.getElementById( 'iframe' ).contentDocument.activeElement );
}, 1000 )
```
and when focus gets lost, it is printing the "html" element of the iframe document.

**Comment by jessegreenberg** [comment]
On the other hand, in Edge, we are getting
> SCRIPT5: SCRIPT5: Access is denied.

EDIT: Edge seems to report this even when focus is working correctly, so it doesn't like https://github.com/phetsims/coulombs-law/issues/74#issuecomment-430768382 either way. NOt a good test.

**Comment by jessegreenberg** [comment]
Moving this issue to scenery since it won't be easy.

**Comment by mbarlow12** [comment]
@arouinfar I just merged the changes to master. Should be all set on both master and the 1.0 branches!

**Comment by jessegreenberg** [comment]
This should be fixed in https://github.com/phetsims/scenery/issues/883, @KatieWoe could you please double check master before we merge into the release branch?

**Comment by KatieWoe** [comment]
@jessegreenberg looks good on all Win 10 and Mac 10.14 browsers when testing resistance in a wire.  The focus stops when it reaches the beginning or end, instead of looping.  Was this the intended fix?  Do you want any testing done one any other platforms or sims?

**Comment by jessegreenberg** [comment]
Thanks for testing @KatieWoe, that seems like a sufficient test. That was the intended fix (though that should only happen in fullscreen mode).

@mbarlow12 I think we are ready to make a maintenance branch in Scenery for this change. Here is the issue: https://github.com/phetsims/scenery/issues/883

Here is the commit to cherry pick into a scenery maintenance branch: https://github.com/phetsims/scenery/commit/89850855af9257d9ea816579ffc417c29b19b7b7

If you run into complications with the cherry-pick please let me know!

**Comment by mbarlow12** [comment]
Great news! Thanks @KatieWoe and @jessegreenberg for the help. I'll create the release first thing tomorrow!

**Comment by mbarlow12** [comment]
Maintenance patch complete and a new rc released:
https://phet-dev.colorado.edu/html/coulombs-law/1.0.0-rc.4/phet/coulombs-law_en_phet.html

**Comment by KatieWoe** [comment]
Looks good on 1.0.0-rc.4

**Comment by mbarlow12** [comment]
Removing the blocking label. @rea-laura feel free to close.
